server-core/engine/src/main/java/io/infectnet/server/engine/core/world/strategy/PathFinderStrategy.java

```diff
@@ -0,0 +1,23 @@
+package io.infectnet.server.engine.core.world.strategy;
```

Comment:
Maybe `pathfinding` would be a better name for this package because we may have other strategies in the future unrelated to pathfinding.